gulpfile.js

```diff
@@ -45,25 +42,8 @@ function vendorAssets() {
     .pipe(gulp.dest(paths.distAssets));
 }
 
-//build content graphviz diagrams into SVG (https://graphviz.org)
-function contentDiagrams() {
-  return gulp.src(paths.srcDiagrams)
-    .pipe(transform("utf8", (mermaidSrc) => {
-      const viz = new Viz(vizRenderOpts);
-      return viz.renderString(mermaidSrc);
-    }))
-    .pipe(rename({extname: ".svg"}))
-    .pipe(gulp.dest(paths.dist));
-}
-
-//content file types which we wish to copy over to dist
-function contentResources() {
-  return gulp.src(paths.srcResources)
-    .pipe(gulp.dest(paths.dist));
-}
```

Comment:
I can no longer have separate build tasks for copying images and rendering .viz diagrams. Their destinations in the output now depend on the page.yml specifying what the localized URLs will be, so these steps how happen at the same time as page rendering.